shared/fs/common/index.stories.js

```diff
@@ -248,7 +248,20 @@ const load = () => {
         />
       </Kb.Box2>
     ))
+    .add('Sync Status', () => (
+      <Kb.Box2 direction="vertical" gap="large" gapStart={true} fullWidth={false} alignItems={'center'}>
+        <SyncStatus status={'awaiting-to-sync'} folder={false} />
+        <SyncStatus status={'awaiting-to-upload'} folder={false} />
+        <SyncStatus status={'online-only'} folder={false} />
+        <SyncStatus status={'synced'} folder={false} />
+        <SyncStatus status={'sync-error'} folder={true} />
+        <SyncStatus status={'uploading'} folder={false} />
+        <SyncStatus status={0.3} folder={false} />
```

Comment:
Just so you know, something like this would also work:

```
['awaiting-to-sync', 'awaiting-to-upload', ......].map(status => <SyncStatus status={status} folder={folder} />)
```

But there's not a lot of props here so what you have is good too. (i.e. not suggesting a change!)